README.md

```diff
@@ -202,6 +203,38 @@ $bodyRequestDef = $schema->getRequestParameters($path, $method);
 $bodyRequestDef->match($requestBody);
 ```
 
+## Validate your Specification
+
+PHP Swagger has the class `MockRequester` with exact the same functionalities of `ApiRequester` class. The only
+difference is the `MockRequester` don't need to request to a real endpoint.
+
+This is particularly usefull if you want to check if your OpenApi specification is returning the expected values. 
```

Comment:
"usefull" -> "useful". Other than that, the OpenApi spec doesn't return anything. Suggestion:

"""This is used to validate request and response against your OpenAPI spec without running any server code."""

I like this addition tough!

Reply:
Thanks :+1: